Src/FluentAssertions/Events/EventRecorder.cs

```diff
@@ -40,6 +42,8 @@ public EventRecorder(object eventRaiser, string eventName, Func<DateTime> utcNow
         /// <inheritdoc />
         public string EventName { get; }
 
+        private EventRaisedOrder EventRaisedOrder { get; }
```

Comment:
🔧 Can't this be a simple field?

Reply:
Done in https://github.com/fluentassertions/fluentassertions/pull/1773/commits/ee0b86ba673e5491c542e9aca797095ad66db2cc.